src/backend/updating-llvm.md

```diff
@@ -2,20 +2,15 @@
 
 <!-- toc -->
 
-The Rust compiler uses LLVM as its primary codegen backend today, and naturally
-we want to at least occasionally update this dependency! Currently we do not
-have a strict policy about when to update LLVM or what it can be updated to, but
-a few guidelines are applied:
-
-* We try to always support the latest released version of LLVM
-* We try to support the "last few" versions of LLVM (how many is changing over
-  time)
-* We allow moving to arbitrary commits during development.
-* Strongly prefer to upstream all patches to LLVM before including them in
-  rustc.
-
-This policy may change over time (or may actually start to exist as a formal
-policy!), but for now these are rough guidelines!
```

Comment:
Why did you remove this sentence? This caution looks useful to me.

Reply:
It's basically repeated, so I keep it in one place... see line 6 of new changes.